modules/social.py

```diff
@@ -1043,6 +1098,71 @@ async def _profile_validate_allow(self, ctx, member: tools.ResolveUser):
         self.validate_allowed_users.append(member.id)
         return await ctx.message.reply(f'{config.greenTick} {member} temporarily added to allowlist')
 
+    @commands.has_any_role(config.moderator, config.eh)
+    @commands.group(name='trivia')
+    async def _trivia(self, ctx):
+        return
+
+    @commands.has_any_role(config.moderator, config.eh)
+    @_trivia.command(name='award')
+    async def _trivia_award(self, ctx, members: commands.Greedy[tools.ResolveUser]):
+        '''Increase the trivia award trophy by one tier for one or more users'''
+        stats = {0: 0, 1: 0, 2: 0, 3: 0, 4: 0, 5: 0, 6: 0, 7: 0, 8: 0, 9: 0}
+        failed = []
+        msg = await ctx.send(f'{config.loading} Processing awards to {len(members)} member(s)...')
+        for m in members:
+            try:
+                newLevel = await self.modify_trivia_level(m)
+                stats[newLevel] += 1
+
+            except IndexError:
+                failed.append(f'{m.mention} ({m.id})')
+
+        embed = discord.Embed(title='Command Completion Stats')
+
+        successList = [(key, value) for key, value in stats.items() if value != 0]
+        embed.description = f'Trivia awards granted to **{len(members) - len(failed)}**.{" List of trophies user(s) now have:" if successList else ""}\n\n'
+        for item in successList:
+            embed.description += f'{self.triviaTrophyEmotes[item[0]]} {self.triviaTrophyIndex[item[0]].replace("-", " ").title()}: {item[1]}\n'
```

Comment:
`self.triviaTrophyIndex[item[0]].replace("-", " ").title()`

This is honestly kinda a bad way to implement this if you ever plan on expanding this. It may be better to have a list/dict/etc of trophy names and just pull from that. You could even condense triviaTrophyIndex and triviaTrophyEmotes into a list of sets together like this. For example:
```py
self.trophyData = [
    ('no-active-trophies', '', ''),
    ('trivia-bronze-1', '<:triviabronze1:1194031669498351656>', ' Trivia Bronze 1'),
    ...
]

Reply:
If you decide to use a set, your `currentLevel = self.triviaTrophyIndex.index(t)` would instead be:
`currentLevel = [n for (n, _, _) in self.trophyData].index(t)`

Reply:
Not as a defense, but if we were to modify this then we'd have to change this codebase wide, as anywhere trophies are humanized it's done this way. Better to do it all at once if we change our naming scheme

Reply:
There is still the option to use an array on tuples without the humanized name (which would have made it a 3-uple)